src/components/shared/VoteCard/VoteCard.vue

```diff
@@ -22,10 +22,12 @@
 <script lang="ts">
 import { Component, Vue, Prop } from "vue-property-decorator";
 import Avatar from "@/components/shared/Avatar/Avatar.vue";
+import Smiley from "../Smiley/Smiley.vue";
 
 @Component({
   components: {
     Avatar,
+    "dw-smiley": Smiley,
```

Comment:
From now on we should add the components as they are, like 

```typescript
@Component({
  components: {	  components: {
    Avatar,	    Avatar,
    Smiley,
}})
```
`dw-` was meant for `dont worry be happy`. And although it could be interesting to have a namespace, these components are not meant to be a reusable library. Passing the component directly prevents us to have to rename everything over and over again.